hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCommits.java

```diff
@@ -303,10 +303,10 @@ public void testThriftExceptionConcurrentCommit() throws TException, Interrupted
 
   @Test
   public void testInvalidObjectException() {
-    TableIdentifier badTi = TableIdentifier.of(DB_NAME, "`tbl`");
+    TableIdentifier badTi = TableIdentifier.of(DB_NAME, "tábl");
```

Comment:
Is this change related to the testing in some way?  This seems unrelated.

Reply:
This is explained in https://github.com/apache/iceberg/pull/12681 as well.
Before Hive 4, the only special character allowed is '/' (so the backtick, '`' is not allowed in the name), but in Hive 4, other special characters, including backtick, are allowed, so for the purpose of an invalid name, I have changed the name used for this test.

Reply:
should it be more explicit in such case?